kiwi/volume_manager/btrfs.py

```diff
@@ -70,6 +70,8 @@ def post_init(self, custom_args):
 
         self.subvol_mount_list = []
         self.toplevel_mount = None
+        self.first_mount = True
```

Comment:
Can we change the logic here ? I think if you construct a volume manager no volumes are mounted,
thus the following feels more natural to me

```python
self.volumes_mounted = False
```

sorry if that's nit picking